src/css/variables.scss

```diff
@@ -0,0 +1,14 @@
+$font_family: 'Open Sans', sans-serif;
+$header_color: #01579B;
+$body_background_color: #EAEBEF;
+$card_background-color: #FEFEFE;
+$font_weight_medium: 400;
+$font_weight_large: 600;
+$font_weight_small: 200;
+$font_size_xlarge: 3em;
+$font_size_large: 2em;
+$font_size_medium: 1.5em;
+$font_size_small: 1em;
+$text_align: center;
+$border: 3px solid #F1C40F ;
+$border_header: 3px solid #039BE5;
```

Comment:
Thank you for setting up the SASS file. It looks really good and is pretty neat!

Reply:
Everything looks great John. Im going to merge.